source/run/include/TG4RunConfiguration.h

```diff
@@ -101,6 +101,14 @@ class TG4RunConfiguration
     // set methods
     void  SetMTApplication(Bool_t mtApplication);
     void  SetParameter(const TString& name, Double_t value);
+    /// Disables internal dispatch to TVirtualMCApplication::ConstructGeometry()
```

Comment:
Please move the member function comments to .cxx (to be consistent with the rest)